airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -422,6 +422,7 @@ def _cast_value(cls, declared_field_types: List, field_name: str, field_value: A
         if target_type_name == "number":
             # do not cast numeric IDs into float, use integer instead
             target_type = int if field_name.endswith("_id") else target_type
```

Comment:
Should it be `int` or `float`, based on [this](https://github.com/airbytehq/airbyte/issues/10463) issue?

Reply:
Should be float